samples/apps/copilot-chat-app/scripts/Start-Backend.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Initializes and runs the Copilot Chat backend.
+
+set -e
+
+ScriptDir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+cd "$ScriptDir/../WebApi"
+
+# Parameters
+Key="$1"
+
+case "$OSTYPE" in
+  darwin*)
+    dotnet dev-certs https --trust ;;
+  msys*)
+    dotnet dev-certs https --trust ;;
+  cygwin*)
+    dotnet dev-certs https --trust ;;
+  linux*)
+    dotnet dev-certs https ;;
+esac
+
+# If key provided, store it in user secrets
+if [ "$Key" != "" ]; then
+    dotnet user-secrets set "Completion:Key" "$Key"
+    dotnet user-secrets set "Embedding:Key" "$Key"
+    dotnet user-secrets set "Planner:AIService:Key" "$Key"
+fi
+
+# Build and run the backend API server
+dotnet build && dotnet run
```

Comment:
Nit: Couldn't we just use dotnet run here?  I think dotnet run implies a dotnet build when no build is available.

The drawback of always having dotnet build is that is takes a few seconds to "build" even though everything is already built and will slow down starts when we already have a build.

Reply:
The benefit of "dotnet build" is better error messaging when it fails.